packages/blade/rollup.config.js

```diff
@@ -61,13 +62,17 @@ const getWebConfig = ({ exportCategory }) => ({
   input: `${inputRootDirectory}/${exportCategory}/index.ts`,
   output: [
     {
-      file: `${outputRootDirectory}/${exportCategory}/index.web.js`,
+      file: `${outputRootDirectory}/${exportCategory}/index.${process.env.NODE_ENV}.web.js`,
       format: 'esm',
       sourcemap: true,
     },
   ],
   external: (id) => id.includes('@babel/runtime'),
   plugins: [
+    pluginReplace({
+      __DEV__: process.env.NODE_ENV !== 'production',
+      preventAssignment: true,
```

Comment:
do we really need this? because we don't do assignments and this will always be set via env vars 🤔

Reply:
Yes, we need to set __DEV__ during build time.

Reply:
the DEV is fine, I was talking about `preventAssignment`, the docs says that we need this to avoid things like `process.env.NODE_ENV` getting reset when the program is running.

Reply:
@kamleshchandnani No, we don't really need this but rollup was logging these warnings so I added it.

<img width="969" alt="Screenshot 2023-07-26 at 3 24 56 PM" src="https://github.com/razorpay/blade/assets/46647141/ff9c36ac-c7c4-4e1b-a76c-d5aca011d293">